corehq/util/test_utils.py

```diff
@@ -281,21 +281,30 @@ def test(self, args=args):
     return add_cases
 
 
-def make_es_ready_form(metadata):
-    # this is rather complicated due to form processor abstractions and ES restrictions
-    # on what data needs to be in the index and is allowed in the index
+def get_form_ready_to_save(metadata, is_db_test=False):
```

Comment:
good eyes, forgot to change it in the skipped test (no impact on build) https://github.com/dimagi/commcare-hq/commit/65e4efe3cad19844e45bd9ca76a18447c76700c2